mscore/plugin/api/selection.h

```diff
@@ -0,0 +1,55 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __PLUGIN_API_SELECTION_H__
+#define __PLUGIN_API_SELECTION_H__
+
+#include "elements.h"
+#include "score.h"
+
+namespace Ms {
+namespace PluginAPI {
+
+//---------------------------------------------------------
+//   Selection
+//    Wrapper class for internal Ms::Selection
+//
+//---------------------------------------------------------
+
+class Selection : public QObject {
+      Q_OBJECT
+      /// Current GUI selections for the score.
+      /// \since MuseScore 3.3
+      Q_PROPERTY(QQmlListProperty<Ms::PluginAPI::Element> selections READ selections)
```

Comment:
Wouldn't it be better just to name this property `elements`, like it is named in the corresponding C++ class? We already have a `Selection` object so it should be absolutely clear that elements behind this property belong to this selection.

Reply:
I was thinking that too even though I felt that 'selections' is more explicit of the truth especially if the Selection exposure is extended on other methods.  Add to that `selection.selections` is just too much of a tongue twister.

I will change.

-Dale